src/emu/epoc/include/epoc/utils/locale.h

```diff
@@ -0,0 +1,671 @@
+/*
+ * Copyright (c) 2020 EKA2L1 Team.
+ * 
+ * This file is part of EKA2L1 project.
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include <cstdint>
+#include <epoc/ptr.h>
+
+namespace eka2l1::epoc {
+
+    enum TLanguage {
```

Comment:
Please format this file to not uses tab

Also rename SLocaleLanguage to our struct naming convention (under_score)

Reply:
I have fixed the tab in previous commits, the naming is changed just now.